src/lib/chips/chip-list.spec.ts

```diff
@@ -19,199 +20,200 @@ describe('MdChipList', () => {
 
-        // Use the spacebar to select the chip
-        chipListInstance._keydown(SPACE_EVENT);
+        // Press the RIGHT arrow
+        chipListInstance._keydown(RIGHT_EVENT);
         fixture.detectChanges();
 
-        expect(firstChip.selected).toBeTruthy();
-        expect(testComponent.chipSelect).toHaveBeenCalledTimes(1);
-        expect(testComponent.chipSelect).toHaveBeenCalledWith(0);
+        // It focuses the next-to-last item
+        expect(manager.focusedItemIndex).toEqual(1);
+      });
 
-        // Use the spacebar to deselect the chip
-        chipListInstance._keydown(SPACE_EVENT);
-        fixture.detectChanges();
+    });
+  });
 
-        expect(firstChip.selected).toBeFalsy();
-        expect(testComponent.chipDeselect).toHaveBeenCalledTimes(1);
-        expect(testComponent.chipDeselect).toHaveBeenCalledWith(0);
-      });
+  describe('InputContainerChipList', () => {
+
+    beforeEach(() => {
+      fixture = TestBed.createComponent(InputContainerChipList);
+      fixture.detectChanges();
+
+      chipListDebugElement = fixture.debugElement.query(By.directive(MdChipList));
+      chipListNativeElement = chipListDebugElement.nativeElement;
+      chipListInstance = chipListDebugElement.componentInstance;
+      testComponent = fixture.debugElement.componentInstance;
+      chips = chipListInstance.chips;
     });
 
-    describe('when selectable is false', () => {
+    describe('keyboard behavior', () => {
       beforeEach(() => {
-        testComponent.selectable = false;
-        fixture.detectChanges();
+        manager = chipListInstance._keyManager;
       });
 
-      it('SPACE ignores selection', () => {
-        let SPACE_EVENT: KeyboardEvent = new FakeEvent(SPACE) as KeyboardEvent;
-        let firstChip: MdChip = chips.toArray()[0];
+      describe('when the input has focus', () => {
 
-        spyOn(testComponent, 'chipSelect');
+        it('DELETE focuses the last chip', () => {
+          let nativeInput = chipListNativeElement.querySelector('input');
+          let DELETE_EVENT: KeyboardEvent = new FakeKeyboardEvent(DELETE, nativeInput) as any;
 
-        // Make sure we have the first chip focused
-        chipListInstance.focus();
+          // Focus the input
+          nativeInput.focus();
+          expect(manager.focusedItemIndex).toBeUndefined();
 
-        // Use the spacebar to attempt to select the chip
-        chipListInstance._keydown(SPACE_EVENT);
-        fixture.detectChanges();
+          // Press the DELETE key
+          chipListInstance._keydown(DELETE_EVENT);
+          fixture.detectChanges();
+
+          // It focuses the last chip
+          expect(manager.focusedItemIndex).toEqual(chips.length - 1);
+        });
+
+        it('BACKSPACE focuses the last chip', () => {
+          let nativeInput = chipListNativeElement.querySelector('input');
+          let BACKSPACE_EVENT: KeyboardEvent = new FakeKeyboardEvent(BACKSPACE, nativeInput) as any;
```

Comment:
Hm, you should be able to do.

```ts
let backspaceEvent = new FakeKeyboardEvent(BACKSPACE, nativeInput) as KeyboardEvent;
```

as in [this example](http://www.typescriptlang.org/play/#src=class%20FakeEvent%20%7B%7D%0D%0A%0D%0A%0D%0Alet%20k%20%3D%20new%20FakeEvent()%20as%20KeyboardEvent%3B%0D%0A%0D%0Adocument.write('Works%20for%20me')%3B)